utils/exploratory_data_analysis.py

```diff
@@ -42,4 +43,32 @@ def get_eda_summary(processed_data_dict: Dict) -> pd.DataFrame:
         ]
         summary[df_name] = results
 
-    return pd.DataFrame.from_dict(summary, orient='index', columns=columns)
\ No newline at end of file
+    return pd.DataFrame.from_dict(summary, orient='index', columns=columns)
+
+
+def plot_time_series(processed_data_dict: Dict):
+    """
+    Parameters:
+        processed_data_dict: Dictionary of previously preprocessed time series data
+    Returns:
+        Plots the times series
+    """
+    checked_data_dict = check_correct_structures(processed_data_dict=processed_data_dict)
+
+    # Create a figure with multiple subplots
+    fig, axs = plt.subplots(len(checked_data_dict), 1, figsize=(20, len(checked_data_dict) * 5))
+    # Iterate over the dictionary
+    for i, (df_name, df) in enumerate(checked_data_dict.items()):
+        # Plot the time series
+        df.plot(x='date', y='value', ax=axs[i])
+        axs[i].set_title(df_name, fontsize=8)
+    plt.show()
+
+
+def get_desc_stats(processed_data_dict: Dict) -> Dict:
```

Comment:
What's the difference between this function and get_eda_summary? The get eda summary get the same metrics as describe() method except the quantiles.